tests/e2e/tests/regression/workbench/workbench-re-cluster.e2e.ts

```diff
@@ -0,0 +1,140 @@
+        await t.click(myRedisDatabasePage.workbenchButton);
+    })
+    .after(async() => {
+        //Delete database
+        await deleteOSSClusterDatabaseApi(ossClusterConfig);
+    })('Verify that user can run commands in Workbench in OSS Cluster DB', async t => {
+        const multipleCommands = [
+            'info',
+            'command',
+            'FT.SEARCH idx *'
+        ];
+        //Send commands
+        await workbenchPage.sendCommandInWorkbench(commandForSend1);
+        await workbenchPage.sendCommandInWorkbench(commandForSend2);
+        //Check that all the previous run commands are saved and displayed
+        await t.eval(() => location.reload());
+        await t.expect(workbenchPage.queryCardCommand.withExactText(commandForSend1).exists).ok('The previous run commands are saved');
+        await t.expect(workbenchPage.queryCardCommand.withExactText(commandForSend2).exists).ok('The previous run commands are saved');
+        //Send multiple commands in one query
+        await workbenchPage.sendCommandInWorkbench(multipleCommands.join('\n'), 0.75);
+        //Check that the results for all commands are displayed
+        for (const command of multipleCommands) {
+            await t.expect(workbenchPage.queryCardCommand.withExactText(command).exists).ok(`The command ${command} from multiple query is displayed`);
+        }
+    });
+test
+    .meta({ env: env.web, rte: rte.sentinel })
+    .before(async t => {
+        await acceptLicenseTermsAndAddSentinelDatabaseApi(ossSentinelConfig, ossSentinelConfig.masters[1].alias);
+        //Go to Workbench page
+        await t.click(myRedisDatabasePage.workbenchButton);
+    })
+    .after(async() => {
+        //Delete database
+        await deleteSentinelDatabaseApi(ossSentinelConfig);
+    })('Verify that user can run commands in Workbench in Sentinel Primary Group', async t => {
+        const multipleCommands = [
+            'info',
+            'command',
+            'FT.SEARCH idx *'
+        ];
+        //Send commands
+        await workbenchPage.sendCommandInWorkbench(commandForSend1);
+        await workbenchPage.sendCommandInWorkbench(commandForSend2);
+        //Check that all the previous run commands are saved and displayed
+        await t.eval(() => location.reload());
+        await t.expect(workbenchPage.queryCardCommand.withExactText(commandForSend1).exists).ok('The previous run commands are saved');
+        await t.expect(workbenchPage.queryCardCommand.withExactText(commandForSend2).exists).ok('The previous run commands are saved');
+        //Send multiple commands in one query
+        await workbenchPage.sendCommandInWorkbench(multipleCommands.join('\n'), 0.75);
+        //Check that the results for all commands are displayed
+        for (const command of multipleCommands) {
+            await t.expect(workbenchPage.queryCardCommand.withExactText(command).exists).ok(`The command ${command} from multiple query is displayed`);
+        }
+    });
```

Comment:
All the tests here are the same, and the diff is the before-test hook and the after one.
Please think about how to make it better instead of copy-paste. There are a few options you can do

Reply:
I  moved the test actions into a separate function in this test file and reused it in each test for each db type

Reply:
Looks much better